src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -190,7 +190,8 @@ export const MousePopulationsModel = types
     "showMaxPoints": false,
     "enableColorChart": true,
     "enableGenotypeChart": true,
-    "enableAllelesChart": true
+    "enableAllelesChart": true,
+    "userChartType": types.union(types.undefined, ChartTypeEnum)
```

Comment:
Two thoughts:
1. Wouldn't `types.maybe(ChartTypeEnum)` be a simpler way of saying the same thing?
2. I suggested putting `userChartType` in the volatile section because it was in the volatile section before. Does putting it in the model have side effects on document storage in the same way it would in CLUE?

Reply:
Shoot, good point about `types.many`.

As for volatile vs. model: I put it back in the model since that's where it was before Sam moved it out, which I assumed he did only to handle defaults more easily. But you're right, I'm not sure if there are side effects, so I'll make it volatile - especially because it seems like there wouldn't be any reason to store this info.

Reply:
Fixed to match your original suggestion!

It is strange that you need to use `undefined as (ChartType | undefined)` instead of `Types.maybe(ChartTypeEnum)`, but I guess this is because volatile state doesn't support getters/setters.